projects/storefrontlib/src/shared/components/anonymous-consents-dialog/anonymous-consent-dialog.component.html

```diff
@@ -1,26 +1,24 @@
 <div
   class="cx-anonymous-consent-dialog"
   [cxFocus]="focusConfig"
-  (esc)="closeModal('Escape clicked')"
+  (esc)="close('Escape clicked')"
 >
-  <div class="modal-content">
+  <div class="cx-dialog-content">
```

Comment:
Can we rework this DOM / css classes a bit more? it's inherited from classic modals, and it's too much I believe. Drop empty DIVs. Drop nested DIVs, avoid heaving both a header and title.